src/main/java/by/exadel/internship/comparator/CityComparator.java

```diff
@@ -0,0 +1,12 @@
+package by.exadel.internship.comparator;
+
+import by.exadel.internship.entity.location.City;
+
+import java.util.Comparator;
+
+public class CityComparator implements Comparator<City> {
```

Comment:
Why do you need both comparator and comparable implementation in City entity?

Reply:
Because I wanted to use Comparable first, and forgot to remove implemented method to that one. Done